tensorflow/stream_executor/stream.cc

```diff
@@ -1886,7 +1886,7 @@ Stream *Stream::GetOrCreateSubStream() {
 
   // Look for the first reusable sub_stream that is ok, dropping !ok sub_streams
   // we encounter along the way.
-  for (int64 index = 0; index < sub_streams_.size();) {
+  for (int64 index = 0, index < static_cast<int64>(sub_streams_.size()); index < end;) {
```

Comment:
```suggestion
  for (int64 index = 0; index < static_cast<int64>(sub_streams_.size());) {
```

But I think we can change type of `index` to be an usigned one

Reply:
updated, I applied the suggested change, as well as making index size_t